setup.py

```diff
@@ -25,7 +25,7 @@ def get_version():
     'google-api-python-client>=1.5.4',
     'six>=1.10.0',  # required by google-api-python-client but not installed by it
     'PyOpenSSL',  # used by oauth2client
-    'httplib2==0.9.2',  # pin this; 0.10.2 causes httplib2.CertificateValidationUnsupported error
+    'httplib2>=0.10.3',  # pin this; 0.10.2 causes httplib2.CertificateValidationUnsupported error
```

Comment:
Seems like we really just want to not use 0.10.2 as opposed to forcing people to be using a version newer than 0.10.2, so I'd prefer `httplib2!=0.10.2`. Also, we should adjust the comment to reflect that (i.e. Skip 0.12.2 because it causes that error).

Reply:
Sounds good. I think 0.10.2 is gone for good but this should be fine just in case.